fetch.bs

```diff
@@ -6160,6 +6188,17 @@ method steps are to <a for=Headers>append</a> (<var>name</var>, <var>value</var>
  <a for=Headers>header list</a>.
 </ol>
 
+<p>The <dfn export for=Headers method><code>getSetCookie()</code></dfn> method steps are:
+
+<ol>
+ <li><p>If <a>this</a>'s <a for=Headers>header list</a> <a for="header list">does not contain</a> `<code>Set-Cookie</code>`, then
```

Comment:
Do we need this step?

Reply:
@annevk says so (I don't understand why though). https://github.com/whatwg/fetch/pull/1346#discussion_r740067545

Reply:
I don't think it's defined behavior what the next step ends up returning otherwise. E.g., things like "in order" expect a list. At the very least we'd have to add "if any" somewhere in there, but I think this is much clearer.

However, this step does need some rewrapping to account for a 100 columns. I also think this PR in general could do with some rebasing.